met-web/src/components/layout/Drawer/CustomDrawer.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import AppBar from '@mui/material/AppBar';
+import { Grid, ListItemButton, List, ListItem, ListItemText, Box } from '@mui/material';
+import { useAppSelector } from '../../../hooks';
+import UserService from '../../../services/UserServices';
+
+
+const list = () => (
+    <Box
+        sx={{
+            background: '#003366',
+            height: '100%',
+            color: 'white',
+            width: 'auto',
+            justifyContent: 'center',
+            alignItems: 'center',
+        }}
+        role="presentation"
+    >
+        <List>
+            {['Engagements', 'Surveys', 'Calendar', 'Reporting'].map((text, index) => (
+                <ListItem key={text}>
+                    <ListItemButton
+                        sx={{
+                            fontWeight: 'bold',
+                            width: '100%',
+                            height: '100%',
+                            alignItems: 'center',
+                            justifyContent: 'center',
+                        }}
+                    >
+                        <ListItemText
+                            sx={{
+                                fontWeight: 'bold',
+                                width: '100%',
+                                height: '100%',
+                                alignItems: 'center',
+                                justifyContent: 'center',
+                            }}
+                            primary={text}
+                        />
+                    </ListItemButton>
+                </ListItem>
+            ))}
+        </List>
+    </Box>
+);
+
+const Drawer = () => {
+    return (
+        <AppBar
+            className="font-BCBold"
+            position="absolute"
+            style={{
+                backgroundColor: '#003366',
```

Comment:
I noticed there was a lot of customization when it came to creating the app bar. Sure we are taking an MUI component, but we are heavily customizing it and it feels we are rebuilding our own app bar. If we absolutely need that, we would do it. But first, we it's better to explore other side bar component solutions out there. The less customization from us the better